cfme/infrastructure/virtual_machines.py

```diff
@@ -124,7 +124,7 @@ class InfraVmDetailsToolbar(InfraGenericDetailsToolbar):
     """Toolbar for VM details differs from All VMs&TemplatesView
     """
     access = Dropdown("Access")
-    power = Dropdown('VM Power Functions')
+    power = Dropdown(text='Power')
```

Comment:
I'm wondering if this change applicable for both 5.10 and 5.11

Reply:
yes @izapolsk 
this change is applicable to both 5.10 and 5.11